openhtf/plugs/__init__.py

```diff
@@ -259,7 +277,8 @@ def plug(update_kwargs=True, **plugs):
     InvalidPlugError: If a type is provided that is not a subclass of BasePlug.
   """
   for a_plug in plugs.itervalues():
-    if not issubclass(a_plug, BasePlug):
+    is_placeholder = isinstance(a_plug, PlugPlaceholder)
+    if not (is_placeholder or issubclass(a_plug, BasePlug)):
```

Comment:
no need for the `is_placeholder` intermediate variable, just collapse these into one line